src/Backend/Core/Js/Components/Meta.js

```diff
@@ -109,18 +81,23 @@ export class Meta {
       // calculate meta
       function calculateMeta (e, element) {
         const title = (typeof element !== 'undefined') ? element.val() : $(this).val()
+        const seoVueRefs = window.backend.seoVue.$refs
 
-        if ($pageTitle.length > 0 && $pageTitleOverwrite.length > 0) {
-          if (!$pageTitleOverwrite.is(':checked')) $pageTitle.val(title)
+        if (seoVueRefs[$pageTitle] !== undefined) {
+          seoVueRefs[$pageTitle].updateValue(title)
         }
 
-        if ($navigationTitle.length > 0 && $navigationTitleOverwrite.length > 0) {
-          if (!$navigationTitleOverwrite.is(':checked')) $navigationTitle.val(title)
+        if (seoVueRefs[$navigationTitle] !== undefined) {
+          seoVueRefs[$navigationTitle].updateValue(title)
         }
 
-        if (!$metaDescriptionOverwrite.is(':checked')) $metaDescription.val(title)
+        if (seoVueRefs[$metaDescription] !== undefined) {
```

Comment:
plsease use `typeof seoVueRefs[$metaDescription] !== 'undefined'` instead, this is not a safe way of checking it